item.go

```diff
@@ -32,6 +32,7 @@ type Item[K comparable, V any] struct {
 	ttl        time.Duration
 	expiresAt  time.Time
 	queueIndex int
+	version    uint64
```

Comment:
Can you add a global option that would enable/disable version tracking? Like here: https://github.com/jellydator/ttlcache/blob/master/options.go#L43

If version tracking is disabled, the version of an item should always be set to `-1`.